fhir/com.b2international.snowowl.fhir.core/src/com/b2international/snowowl/fhir/core/model/codesystem/CodeSystem.java

```diff
@@ -57,6 +67,7 @@
  * @see <a href="https://www.hl7.org/fhir/codesystem.html">FHIR:CodeSystem</a>
  * @since 6.3
  */
+@JsonDeserialize(builder = CodeSystem.Builder.class, using = JsonDeserializer.None.class)
```

Comment:
Not sure if the `using=None` value is needed here. 😕

Reply:
It is there to avoid infinite recursion (deserializer is specified on the FhirResource superclass).